src/Avalonia.Animation/Animatable.cs

```diff
@@ -66,18 +67,20 @@ public Transitions Transitions
         /// <param name="e">The event args.</param>
         protected override void OnPropertyChanged(AvaloniaPropertyChangedEventArgs e)
         {
-            if (e.Priority != BindingPriority.Animation && Transitions != null && _previousTransitions != null)
-            {
-                var match = Transitions.FirstOrDefault(x => x.Property == e.Property);
+            if (_transitions is null || _previousTransitions is null || e.Priority == BindingPriority.Animation) return;
```

Comment:
maybe:
`if ((_transitions is null &&_previousTransitions is null) || e.Priority == BindingPriority.Animation) return;`